dig.go

```diff
@@ -221,16 +231,18 @@ var _graphTmpl = template.Must(
 		Parse(`digraph {
 	graph [compound=true];
 	{{range $g := .Groups}}
-		{{- quote .String}} [shape=diamond label=<{{.Type}}{{.Attributes}}>];
+		{{- quote .String}} [{{.Attributes}}];
 		{{range .Results}}
 			{{- quote $g.String}} -> {{quote .String}};
 		{{end}}
 	{{end -}}
 	{{range $index, $ctor := .Ctors}}
 		subgraph cluster_{{$index}} {
 			{{quote .Name}} [shape=plaintext];
+			{{- if eq .State 1}}color=red;{{end}}
+			{{- if eq .State 2}}color=orange;{{end}}
```

Comment:
You can add a method on `State` and do,

    {{- with .State }}color={{.Color}};{{ end }}

(`with` will execute the block with `.` set to the provided argument if it's non-zero.)

Reply:
Great, this looks a lot nicer 👍